### ISSUE TITLE
The fit method only works with pathfinder

### ISSUE THREAD

**Issue by ferrine** [pymc-devs/pymc-extras#199]
https://github.com/pymc-devs/pymc-experimental/blob/2da3c81bd66122c799bfc318b52d7a7d509ef457/pymc_experimental/inference/fit.py#L16

Isn't it better to move it to PyMC fit or at least delegate other methods to pm.fit?

**Comment by twiecki** [comment]
That's a good idea. I read somewhere that pathfinder wasn't working, was it @fonnesbeck who said that?

But deferring is a good idea.

**Comment by fonnesbeck** [comment]
Yes, I have not gotten it to work at all, outside of very simple models. It probably requires better management of how JAX is used.
